src/components/Stats/Stats.jsx

```diff
@@ -0,0 +1,91 @@
+import styles from './Stats.module.css';
```

Comment:
Gehört die Stats Komponente zur Location? :) Ich mach jetzt mal beide Augen zu, in Zukunft allerdings bitte darauf achten, dass in einem PR immer nur Sachen gemerged werden die auch zu der Story passen.